app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -99,6 +99,11 @@ def export_histories
                         :template => '../../../api/v2/content_view_version_export_histories/index')
     end
 
+    api :GET, "/content_view_versions/export_api_status", N_("true if the export api is pulp3 ready and usable")
+    def export_api_status
+      render json: { api_usable: SmartProxy.pulp_primary.pulp3_repository_type_support?(Katello::Repository::YUM_TYPE) }, status: :ok
```

Comment:
There should be a usage of foreman's deprecation warning in here so that we can remove this in 4.0 (see existing examples in the code) and will be picked up by the deprecations tracker

Reply:
Added